tests/requests/tests.py

```diff
@@ -743,6 +743,135 @@ def test_get_host_suggestion_of_allowed_host(self):
         )
 
 
+class DataUploadMaxMemorySizeGetTests(SimpleTestCase):
+
+    def _test_data_upload_max_memory_size_exceeded(self, content_length_header):
+        request = WSGIRequest({
+            'REQUEST_METHOD': 'GET',
+            'wsgi.input': BytesIO(b''),
+            content_length_header: 3,
+        })
+
+        with self.settings(DATA_UPLOAD_MAX_MEMORY_SIZE=2):
+            with self.assertRaisesMessage(SuspiciousOperation, 'Request data too large'):
+                request.body
```

Comment:
I have added a test that proves that file uploads still work no matter the request body size limit. This limit would only apply if you tried to access request.body directly and not through request.FILES, which I think makes sense.